### DISCUSSION TITLE
Route Manager (Add/Upgrade Vehicles on a route in one screen)

### DISCUSSION THREAD

**Discussion by DanClarkData** [voxeltycoon/issues#910] [General]
The most tedious part of your game is when a business is not getting enough supply and you need to add or upgrade vehicles to increase it (also decrease for oversupply).

In a nutshell, I'd love a window that, tells me how well I'm supplying businesses and allows me to easily add / subtract and upgrade vehicles in order to optimize it.

What I'm suggesting is one window, similar to the one which shows the profitability of routes. I think this window should have the following features:

- Show a list of businesses
- Each business shows a fraction of "last 30 days supply" against "demand". I.e how well am I meeting that demand.
- By each business a list of routes that supply that business. (I never have more than 2 routes per business, 95% it's 1 routes)
- "+" and "-" signs to add and subtract vehicles from route. - vehicles would be automatically sent to garage and deleted or bought from the nearest garage to business and released.
- An indication if vehicle on route are the latest type researched, for particular good.
- An upgrade all button, again selecting the nearest garage to the business.

**Comment by binaryclock03** [comment]
I really like this idea, but maybe instead of by businesses its just a list of your routes. And then give the ability to make route folders so you could classify routes by Routes//Passenger//city1 or Routes//Freight//intercity//city1-city2

**Reply by IDragonfire** [reply]
This is an interesting workaround. The problem is only, you see not where you have issues in your transport network, because goods have different prices.
